config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1400,11 +1400,11 @@ periodics:
       - --gcp-node-image=gci
       - --gcp-nodes=1
       - --gcp-zone=us-west1-b
-      - --ginkgo-parallel=30
+      - --ginkgo-parallel=1
```

Comment:
how many tests need to run serially and how many can run in parallel? In general case I'd prefer two jobs if there are many tests in both categories

Reply:
The scheduler E2E test needs to run in serial. For the remaining It depends on the amount of CPU and memory on the VM and how many workers. We can run more resize tests in parallel if we have more memory and CPU or by reducing the request limits I have picked arbitrarily in the ranges of 100m-750m, 100Mi-500Mi per container. But if we add new tests without keeping the limitation in mind, we are very likely to introduce flaky behavior that can be annoying and time consuming to debug.

That said, having two jobs, one for serial and one for parallel (atleast for cgroupv2) is a very good idea. Running these in parallel in a way that does not introduce flake requires some thinking at experimentation. I'll create an issue for beta to explicitly make parallel E2E tests .

I only noticed the difference in pull job at --ginkgo-parallel=1 vs ci job at --ginkgo-parallel=30 this morning. If I grokked correctly, it means all the pull test runs were always serial. If so, for me, it was encouraging to see majority green results in the few runs we had where parallel was being exercised for the first time, and those 9 or 10 out of 34 that failed were due to test timeout from waiting to be scheduled.

@bobbypage Could you please confirm that `-ginkgo-parallel=1` is the one that is needed as well?

Let me make a quick update to the PR naming the jobs as serial to be explicit about it and create an issue to add parallel job in beta.

Reply:
Updated the description and took an issue for adding parallel test job. Tracking https://github.com/kubernetes/kubernetes/issues/116431

Reply:
can you please add an explicit comment next to test-args with the link to the issue? It will confuse future reviewers that we have serial job that doesn't have [Serial] focus